pkg/postage/listener/listener.go

```diff
@@ -327,19 +321,25 @@ func (l *listener) Listen(from uint64, updater postage.EventUpdater, initState *
 
 	go func() {
 		err := listenf()
+		defer func() {
```

Comment:
few question:
- whats the benefit of a `defer` statement here?
- naked writes are usually not recommended. the synced channel is only read off of during node startup, so if this fails later on you get a deadlock on shutdown